### ISSUE TITLE
what is the /path/to/corpus?

### ISSUE THREAD

**Issue by Dejavu610** [asmitaj08/FuzzingBusyBox_LLM#1]
Dear author,
 
Could you please provide an example of the corpus?

**Comment by asmitaj08** [comment]
It is the path of dir that has busybox elfs. You can download some sample firmware  from camera/router vendor website like : DLink, Netgear, Tenda ,etc. You would have to extract the binary to fetch BusyBox ELF (common tool to extract- binwalk)

**Comment by Dejavu610** [comment]
> It is the path of dir that has busybox elfs. You can download some sample firmware from camera/router vendor website like : DLink, Netgear, Tenda ,etc. You would have to extract the binary to fetch BusyBox ELF (common tool to extract- binwalk)

Thank you for your prompt reply. I see the help information " --input", dest='target_binary', help =' Enter the path of the target binary that you wnat to fuzz', required=True," in the in the fuzz_one_target_main.py. It seems that --input is the path to of busybox elfs and corpus is the path of seed. I wonder what the format of seed should be? Is it possible to provide an example of a gpt generated seed?

Thank you very much.

**Comment by asmitaj08** [comment]
Yes, my bad. Replied in rush, and misread it. It's for the collection of input seeds. It could be either random or LLM generated. Here are samples for some seeds for Busybox `awk` utility generated by LLM . The attached example shows some samples
<img width="590" alt="Screenshot 2024-07-12 at 3 43 16 AM" src="https://github.com/user-attachments/assets/bede5080-da21-4157-9075-f151de445e4c">
. You can write them in different files and store them  as seed in the input_corpus dir.
